rest/taskana-rest-spring-test/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>taskana-rest-spring-test</artifactId>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>Test utils for the taskana rest logic.</description>
+
+  <parent>
+    <groupId>pro.taskana</groupId>
+    <artifactId>taskana-rest-parent</artifactId>
+    <version>5.6.1-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <properties>
+    <sonar.tests/>
+  </properties>
+
+  <dependencies>
+    <dependency>
```

Comment:
please try to sort and group the dependencies.
Example: all taskana (internal) modules should stay together.